src/contracts/apiChangeLog.ts

```diff
@@ -0,0 +1,11 @@
+import { ArmResource } from "./armResource";
+
+export interface ChangeLogPropertyContract {
```

Comment:
Please add jsdocs to all contracts. You can see the documentation here: https://docs.microsoft.com/en-us/rest/api/apimanagement/2019-01-01/apirelease/get#apireleasecontract